### ISSUE TITLE
# decimals for nutrients

### ISSUE THREAD

**Issue by Irondeau79** [BFSSI-Bioinformatics-Lab/nutrition_database#40]
There's specific rounding rules for nutrients that should be applied. The information is found under the table CNFADM_Nutr_Name and variable NUTR_DECIMAL_PLACE.

**Comment by ajverster** [comment]
Some nutrients don't have the column filled out, for example folic acid - should I assume they are zero decimals?

**Comment by Irondeau79** [comment]
Yes that is correct if there is no value then assume zero decimals
